### PR TITLE
fix(Typography): fix lead element and update docs

### PR DESCRIPTION
**Pull request by tujoworker**
This PR does fix some issues with the Lead element and cleans up typography docs and adds better examples.

### CODE REVIEW

File: packages/dnb-eufemia/scripts/prebuild/commitToBranch.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const dotenv = require('dotenv')
 const { isCI } = require('repo-utils')
 const ora = require('ora')
 const path = require('path')
-const simpleGit = require('simple-git/promise') // More info: https://github.com/steveukx/git-js#readme
+const simpleGit = require('simple-git') // More info: https://github.com/steveukx/git-js#readme
```
Comment (langz) [inline packages/dnb-eufemia/scripts/prebuild/commitToBranch.js:10]: Is this change intended? Or even related to "fix lead element and update docs".
Just thoughts.

**Review by langz — APPROVED** [review]
👍

File: packages/dnb-eufemia/scripts/prebuild/commitToBranch.js (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/scripts/prebuild/commitToBranch.js:10]: hehe, no, its not. I will make a new pr to keep it clean. 
The recent security update of this dep, made that we get a warning during visual tests 😝

Comment (langz) [reply packages/dnb-eufemia/scripts/prebuild/commitToBranch.js:10]: Thanks for explaining 🙏

File: packages/dnb-eufemia/src/elements/Lead.js [hunk 2]
```diff
@@ -4,20 +4,8 @@
...
-  ...spacingPropTypes,
-  children: PropTypes.node,
-}
-Lead.defaultProps = {
-  children: null,
-}
+import P from './P'
 
+const Lead = (p) => <P style_type="lead" {...p} />
+Lead.tagName = 'dnb-p--lead'
```
Comment (rodynenko) [inline packages/dnb-eufemia/src/elements/Lead.js:10]: I believe Lead (`.dnb-p--lead`) should have `font-weight: 500;`, now it does not

Comment (tujoworker) [reply packages/dnb-eufemia/src/elements/Lead.js:10]: Thanks! Very good findings 🙌 Its fixed now.
